### DISCUSSION TITLE
Optional Modulation meter in dBr

### DISCUSSION THREAD

**Discussion by Mike86-CZE** [PE5PVB/TEF6686_ESP32#496] [Ideas]
I would like the option to change the modulation scale to dBr units used for modulation power.

**Comment by PE5PVB** [comment]
There is no real scale. Datasheet says: 0 … 100% modulation = 0 … 75 kHz FM dev. 1000 … 2000 [*0.1 %] = 100% … 200% over-modulation range (modulation results are an approximate indication of actual FM dev.)

So it's just an indication.
